README.md

```diff
@@ -17,6 +17,11 @@ The Kubeflow pipelines service has the following goals:
 
 Get started with your first pipeline and read further information in the [documentation](https://github.com/kubeflow/pipelines/wiki).
 
+## Blogs
```

Comment:
nit: Call this "Blog posts about Kubeflow Pipelines" or something similar?

Reply:
I'd go with "Blog posts". If we say just "Blogs", it implies we're recommending entire blog sites that are about Kubeflow. I don't think we need to add "about Kubeflow Pipelines" as people can infer that from the context.

Reply:
done. Thanks Sarah.